requirements.md

```diff
@@ -1,55 +1,52 @@
+This document outlines the requirements and acceptance criteria for the ESGF2-US ESGF-1.5 deployment. ESGF-1.5 is a new version of the US-Dept. of Energy's (US-DOE) ESGF catalog and data archive for CMIP6 and prior ESGF data. (CMIP7 and beyond will instead use the ESGF-NG system.) ESGF-1.5 will allow US-DOE to continue providing access to legacy ESGF data by migrating to new, more easily-maintainable systems. Given that the ESGF-1.5 deployment targets legacy ESGF data, we will minimize the effort and cost to migrate from ESGF-1.0 by making the minimum necessary changes to maintain legacy data access. Ideas for improvements should target ESGF-NG.
 
 ## Changes from ESGF-1.0
 
-- **Discovery** - Replace Apache Solr search indices with Globus Search indices.
-- **Discovery** - (Eventually) Consolidate the separate indices at Lawrence Livermore, Argonne, and Oak Ridge national laboratories into a single US-DOE index.
+- **Discovery** - Replace Apache Solr search indices with a single Globus Search index
 - **Data Access** - Replace legacy HTTP/S servers (for data access) with Globus Connect Server (providing both HTTP/S and bulk data transfer)
 - **Data Access** - Update wget script service to use Globus HTTP/S and begin a deprecation process
 - **Applications** - Replace COG web application with MetaGrid
 - **Applications** - Add intake-esgf command-line interface
 
-## Globus Search indices
+## Globus Search Index
 
-**Phase I:** Initially, each US-DOE laboratory (LLNL, ANL, ORNL) will replace its Solr ESGF index with a distinct Globus Search ESGF index.
-
-**Phase II:** As a separate follow-on activity, the three laboratories will consolidate the three indices into a single US-DOE index that references data wherever it is available.
+Replace Apache Solr search indices at Lawrence Livermore, Argonne, and Oak Ridge National Laboratories with a single consolidated US-DOE Globus Search index referencing all available data replicas.
 
 ### Index contents
 
-In Phase I, each index MUST contain metadata for data stored at the corresponding site (ANL, LLNL, ORNL).
-
-In Phase II, the consolidated index MUST contain metadata for data stored at ANY US-DOE site (ANL, LLNL, ORNL).
+Each dataset will have a single entry in the consolidated index, which will refer to ALL US-DOE locations where the data may be accessed.
 
-In Phase II, each dataset will have a single entry in the consolidated index, and the entry will refer to ALL US-DOE locations where the data may be accessed.
+- **Q:** Will the consolidated index need to refer to any data access options beyond the three US-DOE sites? **A:** YES, at NERSC and potentially other locations.
```

Comment:
Yes, the example I provide above (if I haven't goofed and misunderstood from where the information is coming) lists a record on the LLNL index for data at NCAR.